core/modules/user/tests/user_password_reset.test

```diff
@@ -52,17 +52,17 @@ class UserPasswordResetTest extends BackdropWebTestCase {
     $edit = array('name' => $this->randomName(32));
     $this->backdropPost(NULL, $edit, t('Reset password'));
 
-    $this->assertRaw(t('Sorry, %name is not recognized as a user name or an e-mail address.', array('%name' => $edit['name'])), 'Validation error message shown when trying to request password for invalid account.');
-    $this->assertEqual(count($this->backdropGetMails(array('id' => 'user_password_reset'))), 0, 'No e-mail was sent when requesting a password for an invalid account.');
+    $this->assertRaw(t('Sorry, %name is not recognized as a user name or an email address.', array('%name' => $edit['name'])), 'Validation error message shown when trying to request password for invalid account.');
```

Comment:
Suggestion: change "user name" to "username" to match the rest of the instances.

Reply:
Closing in favour of a new issue for these.